org.civicrm.reqangular/test/services/format-currency.spec.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'common/mocks/data/settings.data',
+  'common/angular',
+  'common/angularMocks',
+  'common/services/api',
+  'common/services/format-currency.service'
+], function (setingsMock) {
+  'use strict';
+
+  describe('FormatCurrencyService', function () {
+    var $httpBackend, $rootScope, FormatCurrencyService;
+
+    beforeEach(module('common.apis', 'common.services'));
+    beforeEach(inject(function (_FormatCurrencyService_, _$httpBackend_, _$rootScope_) {
+      FormatCurrencyService = _FormatCurrencyService_;
+      $httpBackend = _$httpBackend_;
+      $rootScope = _$rootScope_;
+    }));
+
+    afterEach(function () {
+      $httpBackend.flush();
+      $rootScope.$apply();
+    });
+
+    describe("when ' and , are thousand and decimal separators respectively", function () {
+      beforeEach(function () {
+        $httpBackend.whenGET(/entity=Setting/).respond(setingsMock.settings);
+      });
+      describe('addSeperators()', function () {
+        describe('when the amount is without decimal', function () {
+          it('formats the given amount using defined decimal and thousands separator', function () {
+            FormatCurrencyService.addSeperators(6000000000).then(function (result) {
+              expect(result).toEqual("6'000'000'000");
```

Comment:
This is a test for Switzerland?

Reply:
NO! it was just to test if it formats the amount correctly.